storybookjs/storybook
# Fixed prop type validation errors in info addon

Issue: https://github.com/storybooks/storybook/issues/1305

## What I did

Updated PropVal.js to pass its props to child elements in case of arrays and objects.


## How to test

Render a story using `addWithInfo` with a component that receives an array or an object as a prop. Open console to verify you no longer receive errors due to properties validation for PropVal component.
